pdm/pep517/base.py

```diff
@@ -250,7 +250,7 @@ def find_files_to_add(self, for_sdist: bool = False) -> List[Path]:
         If for_sdist is True, will include files like LICENSE, README and pyproject
         Produce a paths list relative to the source dir.
         """
-        return sorted(set(Path(p) for p in self._find_files_iter(for_sdist)))
```

Comment:
This changes the behavior

Reply:
that was a mistake on my part. fixed now.

should have read-

```python
return sorted({Path(p) for p in self._find_files_iter(for_sdist)})
```